gradle.properties

```
@@ -40,7 +40,7 @@ bouncycastleVersion=1.70
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.0.0.xx-SNAPSHOT to pick up maven local published copy
 #cordaApiVersion=5.0.0.xxx-SNAPSHOT
-cordaApiVersion=5.0.0.419-beta+
+cordaApiVersion=5.0.0.420-alpha-1666623635713
```

Once `corda-api` PR is integrated, I think it will be changed to:

```suggestion
cordaApiVersion=5.0.0.420-beta+
```